LLMonFHIR/FHIR Display/MultipleResourceChat.swift

```diff
@@ -0,0 +1,70 @@
+//
+// This source file is part of the Stanford LLM on FHIR project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziOpenAI
+import SwiftUI
+
+
+struct MultipleResourceChat: View {
```

Comment:
The `MultipleResourceChat` looks like a very close copy to the existing view that displays the chat in a single resource. I have documented a general need for a generalized View here: https://github.com/StanfordSpezi/SpeziML/issues/17 . We don't need to do this in the same PR to bring this view to SpeziML, but would be good to ensure that we have a minimal code duplication here. Maybe let's refactor this in a `ResourceChatView` or something similar that contains all the information except the `NavigationStack` and the `navigationTitle` modifier. This would allow us to reuse the view for both use cases and then only need to package that view in a NavigationStack and add a `navigationTitle` modifier.

Reply:
I added a ``` ResourceChatView ``` which handles the chat interface for both single and multiple resources.